tests/api.c

```diff
@@ -15069,6 +15070,23 @@ static void test_wolfSSL_ASN1_TIME_print()
 }
 
 
+static void test_wolfSSL_ASN1_GENERALIZEDTIME_free(){
+    #if defined(OPENSSL_EXTRA)
+    WOLFSSL_ASN1_GENERALIZEDTIME* asn1_gtime;
+    unsigned char nullstr[32];
+
+    XMEMSET(nullstr, 0, 32);
+    asn1_gtime = XMALLOC(sizeof(ASN1_GENERALIZEDTIME), NULL, 
```

Comment:
The result of XMALLOC should be cast to the resulting type.  In this case, it should be cast to (WOLFSSL_ASN1_GENERALIZEDTIME*)

Reply:
I added cast operator.